plugins/pulp_rpm/yum_plugin/util.py

```diff
@@ -197,15 +195,18 @@ def errata_format_to_datetime(datetime_str, msg):
     :rtype: datetime.datetime
     :raises ValueError: if the date and time are in unknown format
     """
-    strptime_pattern = '%Y-%m-%d %H:%M:%S'
+    strptime_patterns = ['%Y-%m-%d %H:%M:%S',
+                         '%Y-%m-%d']
     datetime_str = datetime_str.strip()
     if datetime_str.endswith(' UTC'):
         datetime_str = datetime_str[:-4]
 
-    try:
-        datetime_obj = datetime.datetime.strptime(datetime_str, strptime_pattern)
-    except ValueError:
-        raise PulpCodedValidationException(error_code=error_codes.RPM1007, details=msg,
-                                           expected_format=strptime_pattern)
-
+    for strptime_pattern in strptime_patterns:
+        try:
+            datetime_obj = datetime.datetime.strptime(datetime_str, strptime_pattern)
+            break
+        except ValueError:
+            continue
```

Comment:
Maybe add a debug logging statement that it couldn't parse something here? It's up to you, I'm not sure.